target/linux/ramips/image/mt7621.mk

```diff
@@ -420,6 +420,22 @@ define Device/re350-v1
 endef
 TARGET_DEVICES += re350-v1
 
+define Device/tplink_re650-v1
+  DTS := RE650
```

Comment:
I don't want to stir in some else's soup to much, but since you nicely labelled the device with vendor_model-revision already, why not start to also label the DTS files that way (or even mt7621_tplink_re650-v1.dts ...)
Note that this is without any obligation, I just wanted to propose this as an option.

Reply:
Just discussed that for another device: Keep what you have for now, I will discuss changing that on a more general basis elsewhere.

Reply:
Okay, thanks. I leave it as is. The soc_vendor_device scheme looks better, well, at least to me.
I see from the comments in the other PRs I have at least two things to improve: (1) the wlan led triggers in dts and (2) the SPI frequency for the mtd flash.

There's also a problem regarding the leds. The single eth port has two leds (link, activity). The way it is set now, the link led turns off when the activity led turns on. So it's kinda weird. I expect the link led to be bound to the actual link state and not blink on activity.